src/clippy-tracing/Cargo.toml

```
@@ -0,0 +1,20 @@
+[package]
+name = "clippy-tracing"
+version = "0.1.0"
+authors = ["Amazon Firecracker team <firecracker-devel@amazon.com>"]
+edition = "2021"
+license = "Apache-2.0"
+
+[[bin]]
+bench = false
```

```suggestion
name = "clippy-tracing"
bench = false
```
Locally I had added this as well.